### ISSUE TITLE
Display Terms and Conditions page

### ISSUE THREAD

**Issue by hannavovk** [bcgov/cas-registration#369]
#### Description:

We need a page for IO to agree to Terms and Conditions when they use the Web App. Envisioned to be very similar to the CIIP app. 

Status: 
- Business area and FIN working on the content of the T&C.
- DST to set up placeholder page

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

- Blocked by #370

**Comment by Dianadec** [comment]
Hanna is working on getting the text for the terms and conditions.

**Comment by Dianadec** [comment]
Making a separate ticket for the design.

**Comment by hannavovk** [comment]
Duplicate of #370
